<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TiSIG</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" rowspan="2">
     <widget class="QFrame" name="frame_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item row="2" column="0">
        <widget class="QListWidget" name="listeWidget_layersList">
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDrop</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::MoveAction</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>Gestion des couches</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btn_layerInformation">
            <property name="toolTip">
             <string>Information sur la couche</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../icons/Information.png</normaloff>../icons/Information.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_moveLayerUp">
            <property name="toolTip">
             <string>Monter la couche</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../icons/ArrowUp.png</normaloff>../icons/ArrowUp.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_moveLayerDown">
            <property name="toolTip">
             <string>Descendre la couche</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../icons/ArrowDown.png</normaloff>../icons/ArrowDown.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_zoomOnLayer">
            <property name="toolTip">
             <string>Zoom sur la couche</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../icons/Expand.png</normaloff>../icons/Expand.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_deleteLayer">
            <property name="toolTip">
             <string>Supprimer la couche</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../icons/Trash.png</normaloff>../icons/Trash.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_attributeInformation">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>Informations attributaires</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QTableWidget" name="tableWidget_layerAttributeInformation">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>126</number>
         </attribute>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Nom</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Valeur</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>coucoucoucoucoucocuocuocu</string>
          </property>
          <property name="textAlignment">
           <set>AlignLeading|AlignVCenter</set>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>coucoucoucoucoucou</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>46</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="btn_switchMode2D3D">
         <property name="toolTip">
          <string>Changer de mode</string>
         </property>
         <property name="text">
          <string>2D / 3D</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_zoomIn">
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="toolTip">
          <string>Zoom avant</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/ZoomMore.png</normaloff>../icons/ZoomMore.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_zoomOut">
         <property name="toolTip">
          <string>Zoom arrière</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/ZoomLess.png</normaloff>../icons/ZoomLess.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_zoomFull">
         <property name="toolTip">
          <string>Zoom sur l'emprise totale</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/Expand.png</normaloff>../icons/Expand.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="1" colspan="2">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page_2D">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="2">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_epsg2D">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_5">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Échelle :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLineEdit" name="lineEdit_scale2D">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>EPSG :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="5">
            <widget class="QGraphicsView" name="graphicsView_window2D"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_3D">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="4">
            <widget class="QLineEdit" name="lineEdit_scale3D">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="focusPolicy">
              <enum>Qt::TabFocus</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Échelle :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_epsg3D">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>EPSG :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" colspan="5">
            <widget class="OpenGLcityView" name="openGLWidget_window3D"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_TiSIG">
    <property name="title">
     <string>TiSIG</string>
    </property>
    <addaction name="action_help"/>
   </widget>
   <widget class="QMenu" name="menu_layers">
    <property name="title">
     <string>Couche</string>
    </property>
    <widget class="QMenu" name="menu_addLayers">
     <property name="title">
      <string>Ajouter une couche</string>
     </property>
     <addaction name="action_add2DVectorLayer"/>
     <addaction name="action_add2DRastorLayer"/>
     <addaction name="action_add2DDataFlow"/>
     <addaction name="separator"/>
     <addaction name="action_add3DVectorLayer"/>
     <addaction name="action_add3DRastorLayer"/>
     <addaction name="action_add3DModel"/>
    </widget>
    <addaction name="menu_addLayers"/>
   </widget>
   <addaction name="menu_TiSIG"/>
   <addaction name="menu_layers"/>
  </widget>
  <action name="action_add2DVectorLayer">
   <property name="text">
    <string>Ajouter une couche Vecteur</string>
   </property>
  </action>
  <action name="action_add2DRastorLayer">
   <property name="text">
    <string>Ajouter une couche Raster</string>
   </property>
  </action>
  <action name="action_add2DDataFlow">
   <property name="text">
    <string>Ajouter un flux (WMS, WFS, WMTS)</string>
   </property>
  </action>
  <action name="action_add3DVectorLayer">
   <property name="text">
    <string>Ajouter une couche Vecteur</string>
   </property>
  </action>
  <action name="action_add3DRastorLayer">
   <property name="text">
    <string>Ajouter une couche Raster</string>
   </property>
  </action>
  <action name="action_add3DModel">
   <property name="text">
    <string>Ajouter un modèle 3D</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>Aide</string>
   </property>
  </action>
  <action name="action_Mode_2D">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>--- Mode 2D ---</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_Mode_3D">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>--- Mode 3D ---</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenGLcityView</class>
   <extends>QOpenGLWidget</extends>
   <header>3D/openglcityview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>